src/components/Counter/Counter.stories.tsx

```diff
@@ -30,8 +31,14 @@ DefaultWithTooltip.args = {
   plusButtonText: 'Add by 1',
   fieldNote: 'This is a counter field',
   labelAfter: (
-    <Tooltip buttonText="Select this button to trigger the tooltip">
-      Some text to help with a form field
+    <Tooltip text="Some text to help with a form field">
+      <Button
+        variant="bare"
+        iconPosition="after"
+        iconName="question-mark-circle"
+        aria-label="Hover this button to trigger the tooltip"
+        text="Hover this button to trigger the tooltip"
```

Comment:
Nit - I don't think you need both the `text` and `aria-label` here since they're the same.

Reply:
fixed, thanks